api/task_processor/admin.py

```diff
@@ -14,5 +16,7 @@ class RecurringTaskAdmin(admin.ModelAdmin):
     )
     readonly_fields = ("args", "kwargs")
 
-    def last_run_status(self, instance: RecurringTask) -> str:
-        return instance.task_runs.order_by("-started_at").first().result
+    def last_run_status(self, instance: RecurringTask) -> Optional[str]:
+        return getattr(
+            instance.task_runs.order_by("-started_at").first(), "result", None
+        )
```

Comment:
```suggestion
    def last_run_status(self, instance: RecurringTask) -> Optional[str]:
        if last_run := instance.task_runs.order_by("-started_at").first():
            return last_run.result
        return None
```

Slight preference for this aethetically but have approved regardless.